docs/src/main/tut/symbols.md

```diff
@@ -0,0 +1,27 @@
+#Symbols
+
+Below is a full list of symbols used in cats.  
+
+The `~>`, `⊥` and `⊤` symbols can be imported with `import cats._`.
+
+All other symbols can be imported with `import cats.implicits._`
+
+| Symbol     | Name                   | Typeclass                                                                            | Definition                             |
+| ---------- | ---------------------- | ---------------------------------------------------------------------------------    |--------------------------------------- |
+| `fa |@| fb`| Cartesian builder      | [`Cartesian[F[_]]`](../../../../core/src/main/scala/cats/Cartesian.scala)            | `|@|(fa: F[A])(fb: F[B]): F[(A, B)]`   |
```

Comment:
This is minor and not really important, but it might be slightly cleaner to use something like `{{ site.sources }}/core/src/main/scala/cats/Cartesian.scala`, which is defined [here](https://github.com/typelevel/cats/blob/master/docs/src/site/_config.yml#L6), so jekyll should do the right thing.